Fix typos and improve grammar for aesthetics

ATechnoHazard opened the pull request:
Done as a part of Google Code-in 2017, Cheers!

canihavesomecoffee commented on docs/installing_dependencies.adoc:
```diff
@@ -46,8 +46,8 @@ To install g2p-seq2seq :
 
 
-*I have incorporated above steps into a bash script located inside `install/` directory, that you may simply call using :*
+*I have incorporated above the steps into a bash script located inside `install/` directory. You may call it simply by using :*
```
I think that you should swap around "above" and "the steps", that makes more sense :)

ATechnoHazard commented:
Oh damn, I did this in a jiffy, needed to build my project with the latest
commits, I'll fix and create a new PR asap

On Wed, 29 Nov 2017, 2:55 pm Willem, <notifications@github.com> wrote:

> *@canihavesomecoffee* commented on this pull request.
> ------------------------------
>
> In docs/installing_dependencies.adoc
> <https://github.com/saurabhshri/CCAligner/pull/14#discussion_r153730794>:
>
> > @@ -46,8 +46,8 @@ To install g2p-seq2seq :
>      sudo python setup.py install
>
>
> -*I have incorporated above steps into a bash script located inside `install/` directory, that you may simply call using :*
> +*I have incorporated above the steps into a bash script located inside `install/` directory. You may call it simply by using :*
>
> I think that you should swap around "above" and "the steps", that makes
> more sense :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/saurabhshri/CCAligner/pull/14#pullrequestreview-79779545>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AeSls7tZy8rqTPTrh5IAqD1giXFOdwvAks5s7SL3gaJpZM4QumJI>
> .
>

canihavesomecoffee commented:
@ATechnoHazard you can also push new commits to this pull request :)

saurabhshri commented:
@ATechnoHazard As @canihavesomecoffee you can make a push commits to this branch and that will automatically be added to this PR. You don't need to make a separate PR. I'll wait for the changes recommended by Willem and merge once you add those.

ATechnoHazard commented:
Yeah, I was kinda busy with a few build errors on my ROM, I'll push in some
time


On Wed, 29 Nov 2017, 4:59 pm Saurabh Shrivastava, <notifications@github.com>
wrote:

> @ATechnoHazard <https://github.com/atechnohazard> As @canihavesomecoffee
> <https://github.com/canihavesomecoffee> you can make a push commits to
> this branch and that will automatically be added to this PR. You don't need
> to make a separate PR. I'll wait for the changes recommended by Willem and
> merge once you add those.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/saurabhshri/CCAligner/pull/14#issuecomment-347832949>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AeSls42ju4-_mX3qIolblQcAFB6zoKY2ks5s7UAkgaJpZM4QumJI>
> .
>

ATechnoHazard commented:
Done!

saurabhshri commented:
Nice! Thanks for the PR! :)

ATechnoHazard commented:
Sure thing👍

On Wed, 29 Nov 2017, 7:40 pm Saurabh Shrivastava, <notifications@github.com>
wrote:

> Nice! Thanks for the PR! :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/saurabhshri/CCAligner/pull/14#issuecomment-347871228>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AeSls9t-5tTqU-Ww3U9eA4pmeFHJxDnoks5s7WXvgaJpZM4QumJI>
> .
>
